CONTRIBUTING.adoc

```diff
@@ -0,0 +1,319 @@
+:toc:

```

Comment:
```suggestion
```

Reply:
The first few lines are no longer needed as we are no longer using the Ruby-based framework Awestruct for the build tool in this repo.